test/jdk/java/awt/Frame/FrameResizeTest/FrameResizeTest_1.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (c) 1998, 2023, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+import java.awt.BorderLayout;
+import java.awt.Canvas;
+import java.awt.Color;
+import java.awt.Frame;
+import javax.swing.SwingUtilities;
+
+/*
+ * @test
+ * @bug 4041442
+ * @key headful
+ * @summary Test resizing a frame containing a canvas
+ * @library /java/awt/regtesthelpers
+ * @build PassFailJFrame
+ * @run main/manual FrameResizeTest_1
+ */
+
+public class FrameResizeTest_1 {
+
+    private static final String INSTRUCTIONS = """
+        To the right of this frame is an all-white 200x200 frame.
+
+        This is actually a white canvas component of the frame.
+        The frame itself is red.
+        The red should never show.
+        In particular, after you resize the frame, you should see all white and no red.
+
+        Upon test completion, click Pass or Fail appropriately.
+        """;
+
+    public static void main(String[] args) throws Exception {
+        PassFailJFrame passFailJFrame = new PassFailJFrame.Builder()
+                .title("FrameResizeTest_1 Instructions")
+                .instructions(INSTRUCTIONS)
+                .testTimeOut(5)
+                .rows(9)
+                .columns(45)
+                .build();
+
+        SwingUtilities.invokeAndWait(() -> {
```

Comment:
```suggestion
        EventQueue.invokeAndWait(() -> {
```
Using `EventQueue` is more appropriate for AWT components.

Reply:
Yeah, yet `JFrame` is hidden away inside `PassFailJFrame`.

The test itself uses AWT components, therefore it's reasonable not to use `SwingUtilities`. Functionally, there's no change, so I'm nitpicking.